src/app/nada-consta/nada-consta.service.ts

```
@@ -1,29 +1,49 @@
+  updatedAt: string;
+  createdBy: string;
+  updatedBy: string;
 }
 
-@Injectable({ providedIn: 'root' })
-export class NadaConstaService {
-  private readonly http = inject(HttpClient);
 
+@Injectable({ providedIn: 'root' })
+export class NadaConstaService extends CrudService<NadaConsta, number> {
+  constructor(http: HttpClient) {
```

ESLint

[error] Prefer using the inject() function over constructor parameter injection. Use Angular's migration schematic to automatically refactor: ng generate @angular/core:inject

(@angular-eslint/prefer-inject)